python/rmm/tests/test_rmm.py

```diff
@@ -78,18 +78,22 @@ def test_rmm_modes(dtype, nelem, alloc, managed, pool):
 @pytest.mark.parametrize("nelem", _nelems)
 @pytest.mark.parametrize("alloc", _allocs)
 def test_rmm_csv_log(dtype, nelem, alloc):
```

Comment:
Should probably use a `tmpdir` fixture instead of `tempfile`: https://docs.pytest.org/en/stable/tmpdir.html#the-tmpdir-fixture

Reply:
Do you know if it has the same guarantees as `NamedTemporaryFile`:

> This function operates exactly as TemporaryFile() does, except that the file is guaranteed to have a visible name in the file system (on Unix, the directory entry is not unlinked).

Because IIRC, this guarantee was important for librmm to write to it correctly.

Reply:
Fixed to use `tmpfile` -- looks like it works